### PR TITLE
Powerflex Replication

### PR DESCRIPTION
**Pull request by falfaroc**
# Description
Add the documentation related to enabling `Replication` on PowerFlex.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/618  |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Comment by rajkumar-palani** [comment]
@falfaroc - Demo link is not working, please check. Can you please also correct the typo "Replication" given in the description.

**Review by rajkumar-palani — COMMENTED** [review]
@falfaroc - Demo link is not working, please check. Can you please also correct the typo (update as "Replication") given in the description.

File: content/docs/replication/_index.md
Comment (rajkumar-palani) [inline content/docs/replication/_index.md:39]: Does replication Supports for 1.25 and 1.26 for Powerflex?

File: content/docs/replication/_index.md
Comment (shanmydell) [inline content/docs/replication/_index.md:95]: @falfaroc : K8s n and n-2 matrix support has to be qualified

File: content/docs/replication/deployment/installation.md
Comment (shanmydell) [inline content/docs/replication/deployment/installation.md:78]: Unity doesn't support replication

File: content/docs/replication/_index.md
Comment (shanmydell) [inline content/docs/replication/_index.md:46]: Entire matrix needs to be updated

File: content/docs/replication/deployment/powerflex.md
Comment (shanmydell) [inline content/docs/replication/deployment/powerflex.md:16]: Are these details with respect to dell-csi-extensions needed in documentation?

File: content/docs/replication/deployment/powerflex.md
Comment (shanmydell) [inline content/docs/replication/deployment/powerflex.md:23]: Please rephrase sentence and replace 'your'

**Review by shanmydell — CHANGES_REQUESTED** [review]

**Review by rajkumar-palani — DISMISSED** [review]
Complete the Peer Review on the changes done and get the approval.

File: content/docs/replication/deployment/powerflex.md
Comment (falfaroc) [reply content/docs/replication/deployment/powerflex.md:16]: Keeping it consistent with other drivers' replication documentation.

File: content/docs/replication/deployment/powerflex.md
Comment (rsedlock1958) [inline content/docs/replication/deployment/powerflex.md:30]: Please rewrite this sentence. It does not seem to make sense.

File: content/docs/replication/deployment/powerflex.md
Comment (rsedlock1958) [inline content/docs/replication/deployment/powerflex.md:135]: Replace comma with a full stop (period "."). Initial uppercase on "Let's" to start a new sentence.

File: content/docs/replication/deployment/powerflex.md
Comment (rsedlock1958) [inline content/docs/replication/deployment/powerflex.md:143]: Add "is" after "this"

File: content/docs/replication/deployment/powerflex.md
Comment (rsedlock1958) [inline content/docs/replication/deployment/powerflex.md:207]: Delete "like"

**Review by rsedlock1958 — COMMENTED** [review]
I have reviewed this. Several edits are required.

File: content/docs/replication/deployment/installation.md
Comment (santhoshatdell) [inline content/docs/replication/deployment/installation.md:80]: Similar addition is needed at https://dell.github.io/csm-docs/docs/replication/deployment/storageclasses/#driver-specific-parameters

File: content/docs/replication/deployment/powerflex.md
Comment (santhoshatdell) [inline content/docs/replication/deployment/powerflex.md:5]: Please update the release notes section about new feature.
https://dell.github.io/csm-docs/docs/replication/release/#new-featureschanges

File: content/docs/replication/deployment/powerflex.md
Comment (santhoshatdell) [reply content/docs/replication/deployment/powerflex.md]: @falfaroc : If possible, could you please correct the grammar mistakes in other driver files as well?

File: content/docs/replication/deployment/powerflex.md
Comment (santhoshatdell) [inline content/docs/replication/deployment/powerflex.md:112]: arrayID (or systemID), storagePool and protectionDomain are missing.

File: content/docs/replication/deployment/powerflex.md
Comment (santhoshatdell) [inline content/docs/replication/deployment/powerflex.md:133]: arrayID (or systemID), storagePool and protectionDomain are missing.

**Review by rsedlock1958 — COMMENTED** [review]
I have reviewed this. It looks good. No grammatical edits are required.

File: content/docs/replication/deployment/powerflex.md
Comment (santhoshatdell) [inline content/docs/replication/deployment/powerflex.md:186]: storagePool and protectionDomain are missing.

File: content/docs/replication/deployment/powerflex.md
Comment (santhoshatdell) [inline content/docs/replication/deployment/powerflex.md:164]: storagePool and protectionDomain are missing.

File: content/docs/replication/deployment/powerflex.md
Comment (santhoshatdell) [inline content/docs/replication/deployment/powerflex.md:99]: reclaimPolicy is _Retain_ at other places. Let us be consistent.

File: content/docs/replication/deployment/powerflex.md
Comment (rsedlock1958) [inline content/docs/replication/deployment/powerflex.md:56]: Delete "like"

**Review by rsedlock1958 — COMMENTED** [review]
I have reviewed this. One edit is required.

**Review by santhoshatdell — APPROVED** [review]

**Comment by falfaroc** [comment]
> I have reviewed this. No edits are required.

@rsedlock1958 Please approve if no further changes are needed.

File: content/docs/replication/_index.md
Comment (rajkumar-palani) [inline content/docs/replication/_index.md:42]: check whether 4.11 and 4.10 support for power flex.

File: content/docs/replication/_index.md
Comment (rajkumar-palani) [inline content/docs/replication/_index.md:54]: do we support replication on other model (3.4, 3.5) of power flex arrays?

File: content/docs/replication/_index.md
Comment (rajkumar-palani) [inline content/docs/replication/_index.md:96]: whether powerflex supports 4.11 and 4.10. please double check?

**Review by rajkumar-palani — DISMISSED** [review]
@harshaatdell - can you please review this PR?

File: content/docs/replication/_index.md
Comment (falfaroc) [reply content/docs/replication/_index.md]: Targetted and verified for PowerFlex 3.6 and 4.0.

**Review by rsedlock1958 — APPROVED** [review]
I have reviewed this. No grammatical edits are required.

**Review by santhoshatdell — APPROVED** [review]

**Review by harshaatdell — APPROVED** [review]

**Comment by harshaatdell** [comment]
> @harshaatdell - can you please review this PR?

Done

**Pull request merged**
